src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/EventSourceTests.cs

```diff
@@ -289,7 +296,7 @@ public async Task SeekableTextResponsesErrorsAreLoggedInText()
             Assert.AreEqual("Hello world", errorContentEvent.GetProperty<string>("content"));
         }
 
-        private static async Task<Response> SendRequest(bool isSeekable, Action<MockResponse> setupRequest = null)
+        private async Task<Response> SendRequest(bool isSeekable, Action<MockResponse> setupRequest = null)
```

Comment:
Should this method be called `SendRequestAsync` ?

Reply:
I'll do it as part of https://github.com/Azure/azure-sdk-for-net/issues/6020